tests/validation/tests/v1_api/README.md

```diff
@@ -34,4 +34,3 @@ Examples:
 - test_deployment_with_secret_update()
 - test_monitoring_enable()
 - test_monitoring_update()
```

Comment:
I think we can remove these lines too, no?

Reply:
Removed!